src/realm.h

```diff
@@ -1152,6 +1172,15 @@ RLM_API bool realm_find_primary_key_property(const realm_t*, realm_class_key_t c
  */
 RLM_API bool realm_get_num_objects(const realm_t*, realm_class_key_t, size_t* out_count);
 
+/**
+ * Get the number of versions found in the Realm file.
+ *
+ * @param out_count A pointer to a `size_t` that will contain the number of
+ *                  versions, if successful.
+ * @return True if the number of versions could be found.
```

Comment:
I suppose the only way this can happen is if the realm file is invalid in some way, in which case I think the return value should just be "True if no exception occurred".